src/codegen/codegen_cpp_visitor.cpp

```diff
@@ -4121,7 +4119,7 @@ void CodegenCVisitor::print_net_receive_kernel() {
     }
 
     printer->add_newline(2);
-    printer->fmt_start_block("static inline void {}({}) ", name, get_parameter_str(params));
+    printer->fmt_start_block("static inline void {}({})", name, get_parameter_str(params));
```

Comment:
Looking at the code coverage report I saw that the code corresponding to `info.artificial_cell` is not covered and it should be easy to do by taking as an example: https://github.com/neuronsimulator/nrn/blob/master/test/coreneuron/mod%20files/invlfire.mod

Reply:
This can be done in another PR. #957 can stay open